includes/Core/Authentication/Setup.php

```diff
@@ -63,6 +63,15 @@ class Setup {
 	 */
 	protected $google_proxy;
 
+	/**
+	 * Proxy support URL.
+	 *
+	 * @since 1.48.0
+	 *
+	 * @var Proxy_Support_Link_Url
```

Comment:
Similar here, we should be clear/accurate about the version number, and what this `@var` declaration means.